src/evict/evict_lru.c

```diff
@@ -930,7 +990,7 @@ __wt_evict_file_exclusive_on(WT_SESSION_IMPL *session)
      * any existing LRU eviction walk for the file.
      */
     (void)__wt_atomic_addv32(&cache->pass_intr, 1);
-    WT_WITH_PASS_LOCK(session, ret = __evict_clear_walk(session));
+    WT_WITH_PASS_LOCK(session, ret = __evict_clear_walk(session, true));
```

Comment:
What is the motivation for saving position here?

Reply:
Discussed offline